src/python/WMCore/MicroService/MSTransferor/MSTransferorError.py

```diff
@@ -0,0 +1,85 @@
+"""
+File       : MSTransferorError.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: MSTransferorError represents MSTransferor errors
+"""
+
+# system modules
+import json
+
+# WMCore modules
+from WMCore.WMException import WMException
+
+
+# MSTransferor error codes
+MSPILEUP_GENERIC_ERROR = 1
+MSPILEUP_STORAGE_ERROR = 2
+
+
+class MSTransferorError(WMException):
+    """
+    MSTransferorError represents generic MSTransferor error
+    """
+    def __init__(self, data, msg=""):
```

Comment:
I noticed this class signature differs from the parent class. Is it intentional and was there any strong reason for that? Otherwise, perhaps we could keep it consistent with the parent class.

Honestly, I expected to see complaints in the pylint check

Reply:
I created MSTransferorError from MSPileup one, but ok, modified to align with parent class.